### PR TITLE
Add ROS-Noetic Debian Install Instructions

### PR DESCRIPTION
**Pull request by amm98d**
Tested this on Ubuntu 20.04 with ROS Kinetic and it worked perfectly. These instructions are desperately needed for new beginners, would be really awesome if they could be incorporated 🙂.

### CODE REVIEW

File: ros_installing/tutorial.md [hunk 1]
```diff
@@ -92,6 +92,12 @@ sudo apt-get install ros-kinetic-gazebo-ros-pkgs ros-kinetic-gazebo-ros-control
 sudo apt-get install ros-indigo-gazebo-ros-pkgs ros-indigo-gazebo-ros-control
 ~~~
 
+* [ROS Noetic](http://ros.org/wiki/noetic):
```
Comment (chapulina) [inline ros_installing/tutorial.md:95]: Do you mind adding it up top? I believe the versions are listed in reverse chronological order.

**Review by chapulina — COMMENTED** [review]
Thanks! It would also be nice to add Melodic. And we can probably also remove Lunar and Indigo, which are [EOL](http://wiki.ros.org/Distributions). We can do this in a separate PR if it's too much to add here.

File: ros_installing/tutorial.md (hunk 1 above)
Comment (amm98d) [reply ros_installing/tutorial.md:95]: Sure thing! I'll also add melodic as I also tested it last night.

**Comment by amm98d** [comment]
Also opening a separate PR to remove EOL versions.

**Review by chapulina — APPROVED** [review]
Thanks!

**Pull request merged**